XenAdmin/Controls/Ballooning/ShinyBar.cs

```diff
@@ -244,7 +250,7 @@ protected void DrawGrid(Graphics g, Rectangle barArea, double bytesPerPixel, dou
             using (Pen pen = new Pen(Grid))
             {
                 bool withLabel = true;
-                for (long x = 0; x <= max; x += incr)
+                for (double x = 0; x <= max; x += incr)
                 {
                     // Tick
                     int pos = barArea.Left + (int)((double)x / bytesPerPixel);

```

Comment:
Unnecessary cast of `x` to double. Same further up in line 246 with `incr` (but it won't let me add a comment there).

Reply:
If `max` is a double, the same should apply to `x` and `incr` since `long`s cannot handle as large a set of numbers.

If anything, I think `incr` should be a `double` - that was my intention at least, I just missed it